change -use_ms1_traces to -enable_ms1 in OpenSwathWorkflow docs

jcharkow opened the pull request:
# Description
To my knowledge, -use_ms1_traces parameter has been deprecated and replaced with -enable_ms1. Update the documentation to reflect this. 

Please include a summary of the change and which issue is fixed.

# Checklist:
- [X] Make sure that you are listed in the AUTHORS file
- [X] Add relevant changes and new features to the CHANGELOG file
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes
- [X] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

jpfeuffer commented:
What about https://github.com/OpenMS/OpenMS/blob/e66b2c45ae29fab5fb97e4c4d0e69e75e1849b91/src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp#L480

Maybe we should change every occurrence of use_ms1_traces to make sure that internal parameter names and command line parameters match. Seems like they have a 1:1 correspondence in this case.

jcharkow commented:
Yes I agree, I'll look for other instances in which use_ms1_traces is used.

jcharkow commented:
I replaced all occurrences of `use_ms1_traces` with `enable_ms1` in the documentation and I also replaced the variable occurrences to stay consistent (`use_ms1_traces` and `use_ms1_traces_` with `enable_ms1`and `enable_ms1_`). I don't think this should cause any issues however I am not too experienced with C++.
